Assets/HoloToolkit/Common/Scripts/Singleton.cs.meta

```diff
@@ -1,8 +1,9 @@
 fileFormatVersion: 2
-guid: b0ed29a95096a25448535f8df9cf0423
-timeCreated: 1455735878
+guid: 3ac3ff05b2d583947859298e1ee17955
```

Comment:
please revert this file. May have unintended consequences in other people's projects.